modules/virt-adding-kernel-arguments-enable-iommu.adoc

```diff
@@ -50,9 +50,23 @@ $ oc create -f 100-worker-kernel-arg-iommu.yaml
 
 .Verification
 
-* Verify that the new `MachineConfig` object was added.
+.  Verify that the new `MachineConfig` object was added by entering the following command:
```

Comment:
```suggestion
.  Verify that the new `MachineConfig` object was added by entering the following command and observing the output:
```

I would recommend standardizing these and calling out the need to observe the output as part of the command step. I would also add an example output here if possible, similar to the next step